test/export/test_export.py

```diff
@@ -64,7 +64,7 @@
     from torchrec.sparse.jagged_tensor import KeyedJaggedTensor
 
     HAS_TORCHREC = True
-except ImportError:
+except (ImportError, AttributeError):
```

Comment:
Why the change? Are we actually importing torchrec correctly in CI?

Reply:
Yeah sorry this is my local thing. Not needed.